eng/pipelines/templates/jobs/new.smoke.tests.yml

```diff
@@ -45,83 +45,8 @@ jobs:
-          OSVmImage: "MMSUbuntu20.04"
-          TestTargetFramework: netcoreapp3.1
-          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
-          ArmTemplateParameters: $(azureCloudArmParameters)
-          Location: "westus"
-        Windows_NetCoreApp (AzureCloud):
-          Pool: "azsdk-pool-mms-win-2019-general"
-          OSVmImage: "MMS2019"
-          TestTargetFramework: netcoreapp3.1
-          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
-          ArmTemplateParameters: $(azureCloudArmParameters)
-          Location: "westus"
-        Windows_NetFramework (AzureCloud):
-          Pool: "azsdk-pool-mms-win-2019-general"
-          OSVmImage: "MMS2019"
-          TestTargetFramework: net461
-          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
-          ArmTemplateParameters: $(azureCloudArmParameters)
-          Location: "westus"
-        MacOs (AzureCloud):
-          Pool: Azure Pipelines
-          OSVmImage: "macOS-10.15"
-          TestTargetFramework: netcoreapp3.1
-          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
-          ArmTemplateParameters: $(azureCloudArmParameters)
-          Location: "westus"
-        Windows_Net50 (AzureCloud):
-          Pool: "azsdk-pool-mms-win-2019-general"
-          OSVmImage: "MMS2019"
-          TestTargetFramework: net6.0
-          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
-          ArmTemplateParameters: $(azureCloudArmParameters)
-          Location: "westus"
-        Windows_NetCoreApp (AzureUSGovernment):
-          Pool: "azsdk-pool-mms-win-2019-general"
-          OSVmImage: "MMS2019"
-          TestTargetFramework: netcoreapp3.1
-          SubscriptionConfiguration: $(sub-config-gov-test-resources)
-          ArmTemplateParameters: $(azureUSGovernmentArmParameters)
-          Location: "usgovarizona"
-        Windows_NetFramework (AzureUSGovernment):
-          Pool: "azsdk-pool-mms-win-2019-general"
-          OSVmImage: "MMS2019"
-          TestTargetFramework: net461
-          SubscriptionConfiguration: $(sub-config-gov-test-resources)
-          ArmTemplateParameters: $(azureUSGovernmentArmParameters)
-          Location: "usgovarizona"
-        Windows_NetCoreApp (AzureChinaCloud):
-          Pool: "azsdk-pool-mms-win-2019-general"
-          OSVmImage: "MMS2019"
-          TestTargetFramework: netcoreapp3.1
-          SubscriptionConfiguration: $(sub-config-cn-test-resources)
-          ArmTemplateParameters: $(azureChinaCloudArmParameters)
-          Location: "chinaeast2"
-        Windows_NetFramework (AzureChinaCloud):
-          Pool: "azsdk-pool-mms-win-2019-general"
-          OSVmImage: "MMS2019"
-          TestTargetFramework: net461
-          SubscriptionConfiguration: $(sub-config-cn-test-resources)
-          ArmTemplateParameters: $(azureChinaCloudArmParameters)
-          Location: "chinaeast2"
```

Comment:
Some libraries actually do different between Framework and Core. I wonder if it's worth keeping these and only these two. Thoughts?

Reply:
Though, probably just AzureCloud since propagation of new api-versions to sovereign clouds trails AzurePublic, and some sovereign clouds like Mooncake may not get all of them.

Reply:
We're not doing live tests anymore [ref](https://github.com/Azure/azure-sdk-for-net/pull/25673#discussion_r830390592), so cloud wouldn't matter